src/ExpressiveInstaller/Resources/src/CallableFactory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use Interop\Container\ContainerInterface;
```

Comment:
the test (see https://github.com/zendframework/zend-expressive-skeleton/blob/master/test/ExpressiveInstallerTest/ContainersTest.php#L80) at the moment requires explicitely `Container\Interop\ContainerInterop`.

Another option would be to change the test and ask that `$container` is instance either of `Container\Interop\ContainerInterop` or `Psr\Container\ContainerInterface`